### PR TITLE
improved appearance of all character models

### PR DESCRIPTION
**Pull request by akcoles**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -72,7 +73,7 @@ define([
   /**
    * Create the game, bud
    */
-  var game = new Phaser.Game(WIDTH, HEIGHT, Phaser.AUTO, 'typonomy', state);
+  var game = new Phaser.Game(WIDTH, HEIGHT, Phaser.CANVAS, 'typonomy', state);
```
Comment (nmacmunn) [inline js/main.js]: Why this change?

**Comment by nmacmunn** [comment]
Looks awesome. I think we should figure out how to scale down the rocket because it seems way bigger than the other sprites right now, but that can happen later. Only change I'm going to make is to remove `js.main.js` which doesn't seem relevant to the PR. I'll push it directly to master and leave this PR for you to close at your leisure.

File: js/main.js (hunk 1 above)
Comment (akcoles) [inline js/main.js]: One of the issues I was dealing with had to do with the renderer, I ended up changing the images so it didn't matter though so I should have changed this back to auto.  good catch
